tests/common/dualtor/tor_failure_utils.py

```diff
@@ -27,15 +27,16 @@ def shutdown_tor_bgp(duthost, shutdown_all=True):
         bgp_neighbors = duthost.get_bgp_neighbors()
         up_bgp_neighbors = [k.lower() for k, v in bgp_neighbors.items() if v["state"] == "established"]
         if shutdown_all and up_bgp_neighbors:
-            logger.info("Shutdown BGP sessions on {}".format(duthost.hostname))
-            duthost.shell("config bgp shutdown all")
+            logger.info("Kill bgpd process on {}".format(duthost.hostname))
+            duthost.shell("pkill -9 bgpd")
```

Comment:
I think it depends on how we kill the container - if we use `docker stop` or `systemctl stop` then the bgpd would get a catchable signal such as `SIGTERM` or `SIGINT`, which prompts bgpd to explicitly shutdown the BGP sessions - therefore, the T1 would have no routes to the ToR and the IP-in-IP tunnel from standby->active would not be functional (so we would expect some traffic drops).

Using `SIGKILL` means the process does not have a chance to cleanup after itself, so traffic would not be interrupted until either the BGP hold time on the T1 expires or the container autorestarts (although I did not see any traffic disruptions for container autorestart).

All the tickets I have seen in production have been for `SIGKILL` or some other unexpected signal, so I think this would be the most accurate test scenario.